packages/dds/test-dds-utils/src/clientLoading.ts

```diff
@@ -72,7 +72,7 @@ export function createLoadData(
 ): ClientLoadData {
 	const compressor = client.dataStoreRuntime.idCompressor;
 	return {
-		minimumSequenceNumber: client.dataStoreRuntime.deltaManagerInternal.lastSequenceNumber,
+		minimumSequenceNumber: client.dataStoreRuntime.deltaManagerInternal.minimumSequenceNumber,
```

Comment:
this value was incorrect, and didn't match the value seen by dds who look at the delta manager directly.